pkg/util/subnet/const.go

```diff
@@ -5,6 +5,5 @@ package subnet
 
 // Network security group constants
 const (
-	NSGControlPlaneSuffix = "-controlplane-nsg"
-	NSGNodeSuffix         = "-node-nsg"
+	NSGSuffix = "-nsg"
```

Comment:
The changes in pkg/util/subnet are going to break dynamic validation for clusters on the <4.5 architecture.  This is going to need more thought.

Reply:
i.e. the problem is NetworkSecurityGroupID

Reply:
We need backward compatibility for NSG and LB for all the functions to cover both architectures.

Reply:
Options: Architecture version flag in the documents, where we version architectures. And some kind of function where we decide what is what